docs/index.rst

```diff
@@ -11,4 +11,5 @@ Welcome to Cider's documentation!
     user-guide
     cider-sharing
     development-specification
-    CiderAllocator-use-guide
\ No newline at end of file
+    CiderAllocator-use-guide
+    logger
```

Comment:
suggest putting these pages in an existing folder or page (Cider Developer Guide) better?